test/tests/performance/pbs_qstat_performance.py

```diff
@@ -129,8 +129,7 @@ def submit_and_stat_jobs(self, number_jobs):
         """
         self.submit_jobs(TEST_USER1, number_jobs)
         for query in self.qstat_query_list:
-            self.assertTrue(self.compute_elapse_time(
-                query) < 0, "qstat command failed")
+            self.compute_elapse_time(query)
```

Comment:
why did you remove call to `assertTrue()`?

Reply:
We are calling an `assertTrue()` at the end of `compute_elalpse_time()` . `assertTrue()` returns None. In Python 2, we could compare None with  0, in Python 3 we get an error. 
[nehap@neha20 pbspro]$ python
Python 2.7.5 (default, Jun 20 2019, 20:27:34) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> a = None
>>> a < 0
True
>>> 
[nehap@neha20 pbspro]$ python3
Python 3.6.8 (default, Apr 25 2019, 21:02:35) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a = None
>>> a < 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '<' not supported between instances of 'NoneType' and 'int'
>>> 

Since we are already logging "`"Qstat command with option : " + query + " Failed"`" in `compute_elapse_time()`, I think we don't need to log the failure message again.